compression/rlecoding_test.go

```diff
@@ -0,0 +1,161 @@
+package compression_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/TheAlgorithms/Go/compression"
+)
+
+func TestCompression_RLEncode(t *testing.T) {
```

Comment:
<samp> Convert all the names to simple UpperCamelCase. </samp>

Reply:
Done!